src/api/vdom.js

```diff
@@ -321,6 +321,8 @@ export function element(tname, attrs, ...chren) {
       ch();
     } else if (ctype === 'string' || ctype === 'number') {
       newText(ch);
+    } else if (ctype === 'object') {
```

Comment:
Yeah, not sure why I used `'object'`. Possibly because I was testing object literal traversal at some point? Will change.